tools/run_tests/xds_k8s_test_driver/tests/authz_test.py

```diff
@@ -47,12 +48,14 @@ class AuthzTest(xds_k8s_testcase.SecurityXdsKubernetesTestCase):
     }
 
     @staticmethod
-    def isSupported(config: skips.TestConfig) -> bool:
-        if config.client_lang in ['cpp', 'python']:
-            return config.version_ge('v1.44.x')
-        elif config.client_lang in ['java', 'go']:
-            return config.version_ge('v1.42.x')
-        return False
+    def is_supported(config: skips.TestConfig) -> bool:
+        # Per "Authorization (RBAC)" in
+        # https://github.com/grpc/grpc/blob/master/doc/grpc_xds_features.md
+        if config.client_lang in _Lang.CPP | _Lang.PYTHON:
+            return not config.version_lt('v1.44.x')
+        elif config.client_lang in _Lang.GO | _Lang.JAVA:
+            return not config.version_lt('v1.42.x')
+        return True
```

Comment:
I would recommend to use allowlist pattern. When a new test come online, people will need to enable it one language at a time. When a new language (Node) is added, people can slowly enable tests that are passing consistently for the new language in each release.

Reply:
I'm a bit worried we're not going to run something on accident. Maybe we could add do something specific for node? 

```py
if config.client_lang is _Lang.NODE:
    return False 
```

Or remove node from the list of languages for now, and do

```py
if config.client_lang is _Lang.UNKNOWN:
    return False 
```

cc @murgatroid99

Reply:
Spoke with Michael. He's fine with specifically excluding node for unsupported tests, when applicable.

Also I noticed that when you don't specify `is_supported`, everything is enabled by default. So it's an argument for having the same behavior in tests that have `is_supported` implemented as well.